$MirOS: ports/emulators/qemu/patches/patch-Makefile_target,v 1.4 2005/11/15 12:25:08 tg Exp $
--- Makefile.target.orig	Sun Sep  4 17:11:09 2005
+++ Makefile.target	Mon Oct  2 21:55:30 2006
@@ -17,7 +17,9 @@ ifdef CONFIG_USER_ONLY
 VPATH+=:$(SRC_PATH)/linux-user
 DEFINES+=-I$(SRC_PATH)/linux-user -I$(SRC_PATH)/linux-user/$(TARGET_ARCH)
 endif
-CFLAGS=-Wall -O2 -g -fno-strict-aliasing
+CFLAGS?=-O2 -g
+CFLAGS+=-fno-strict-aliasing -Dasm=__asm__ -Wall -Wextra -Werror -std=c99 -Wno-sign-compare -Wno-unused
+HOST_CC?=$(CC)
 #CFLAGS+=-Werror
 LDFLAGS=-g
 LIBS=
@@ -151,6 +153,8 @@ endif
 #########################################################
 
 DEFINES+=-D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
+
+#DEFINES+=-DDEBUG_GDB -DDEBUG -DDEBUG_KBD -DDEBUG_MOUSE
 LIBS+=-lm
 ifndef CONFIG_USER_ONLY
 LIBS+=-lz
@@ -269,7 +273,10 @@ AUDIODRV += sdlaudio.o
 endif
 ifdef CONFIG_OSS
 AUDIODRV += ossaudio.o
+ifdef CONFIG_OSS_LIBRARY
+LIBS += -lossaudio
 endif
+endif
 
 pc.o: DEFINES := -DUSE_SB16 $(DEFINES)
 
@@ -427,7 +434,7 @@ clean:
 
 install: all 
 ifneq ($(PROGS),)
-	install -m 755 -s $(PROGS) "$(bindir)"
+	install -m 755 -s $(PROGS) $(PREFIX)/bin/
 endif
 
 ifneq ($(wildcard .depend),)
